project/authentication/authentication.py

```diff
@@ -99,8 +99,7 @@ def cw_register(request):
                 user = authenticate(username=username, password=password)
 
                 account = Account(user=user)
-                if not settings.CLOSED_BETA:
-                    account.beta_access = True
+                account.beta_access = True
```

Comment:
Let's remove this field entirely, since there will no longer be a `beta_access` field in the `Account` model.